Core/Core/Dashboard/K5/View/ImportantDates/K5ImportantDatesView.swift

```diff
@@ -0,0 +1,94 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2021-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import SwiftUI
+
+public struct K5ImportantDatesView: View {
+    @ObservedObject var viewModel: K5ImportantDatesViewModel
+
+    public init(viewModel: K5ImportantDatesViewModel) {
+        self.viewModel = viewModel
+    }
+
+    public var body: some View {
```

Comment:
I would split this body into smaller properties, the indentation level is too high.